README.md

```diff
@@ -255,6 +255,67 @@ useminPrepare: {
 
 The given steps or post-processors may be specified as strings (for the default steps and post-processors), or as an object (for the user-defined ones).
 
+### warnMissing
+
+Type: 'boolean'
+Default: `nil`
+
+Whether usemin should issue a warning when it can't find one of the sources
+referenced in a block. For backwards compatibility, this is off by default.
+
+### resolveSource
+
+Type: 'function'
+Default: `nil`
```

Comment:
`resolveSource` defaults to `nil` which seems consistent with README where these others options also defaults to `nil`:

```
### dest

Type: 'string'  
Default: `nil`

Base directory where the transformed files should be output.

### root

Type: 'string' or 'Array'  
Default: `nil`
```